kustomize/commands/localize/localize.go

```diff
@@ -74,6 +103,12 @@ kustomize localize https://github.com/kubernetes-sigs/kustomize//api/krusty/test
 Cannot specify for remote targets, as scope is by default the containing repo.
 If not specified for local target, scope defaults to target.
 `)
+	cmd.Flags().BoolVar(&f.noVerify,
+		"noVerify",
```

Comment:
Since all flags/options in Kustomize currently use the dash standard, this new one should as well (as specified in the KEP):

```suggestion
		"no-verify",
```